src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,104 @@
+# Genetic Descriptor
+
+Support for genetic descriptors was developed as a [BIDS Extension
+Proposal](../06-extensions.md#bids-extension-proposals).
+The extension was primarily developped by Cyril Pernet and Clara Moreau with
+contributions from Tom Nichols and Jessica Turner.
+
+Genetic data are typically stored in dedicated repositories,
+separate from imaging data.
+A genetic descriptor links a BIDS dataset to associated genetic data,
+potentially in a separate repository,
+with details of where to find the genetic data and the type of data available.
+
+## Dataset Description
+
+Genetic descriptors are encoded as an additional, OPTIONAL entry in the
+[`dataset_description.json`](../03-modality-agnostic-files.md#dataset_descriptionjson)
+file.
+
+Datasets linked to a genetic database entry include the following REQUIRED or OPTIONAL
+`dataset_description.json` keys (a dot in the key name denotes a key in a subdictionary):
+
+| Field name           | Definition                                                                     |
+|----------------------|--------------------------------------------------------------------------------|
+| Genetics.Dataset     | REQUIRED. URI where data can be retrieved.                                     |
+| Genetics.Database    | OPTIONAL. URI of database where the dataset is hosted.                         |
+| Genetics.Descriptors | OPTIONAL. List of relevant descriptors (*e.g.*, journal articles) for dataset. |
+
+Example:
+
+```JSON
+{
+  "Name": "Human Connectome Project",
+  "BIDSVersion":  "1.3.0",
+  "License": "CC0",
+  "Authors": ["1st author", "2nd author"],
+  "Funding": ["P41 EB015894/EB/NIBIB NIH HHS/United States"],
+  "Genetics": {
+     "Dataset": "https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001364.v1.p1",
+     "Database": "https://www.ncbi.nlm.nih.gov/gap/",
+     "Descriptors": ["https://doi.org/10.1016/j.neuroimage.2013.05.041"]
+     }
+}
+```
+
+## Subject naming and Participants file
+
+If the same participants have different identifiers in the genetic and imaging datasets,
+the column `GeneticID` SHOULD be added to the `participants.tsv` file to associate
```

Comment:
@CPernet Is it very important to use `GeneticID` instead of `genetic_id`? It is inconsistent with TSV column naming conventions elsewhere in the spec.

Reply:
we can use whatever we want, but this is used twice in the spec and it appears in the example, we just need to be consistent, now I'm not bothered if you prefer `GeneticID` or `genetic_id` pick what you think works best / is consistent with other specs naming

Reply:
genetic_id everywhere pushed